protocol/0073-LIMN-limited_network_life.md

```diff
@@ -334,3 +334,60 @@ Can deposit and withdraw funds to/from ERC20 asset loaded from checkpoint (<a na
 1. The market is still in "pending" state.
 1. The market becomes enaced when the enactment time is passed.
 1. Other parties can trade on the market, and become continuous.
+
+
+
+## Test case 14: Market with trading terminated is not restored, collateral moved correctly 
+1. Set LP fee distribution time step to non-zero value.
+1. Propose, enact, trade in the market, close out distressed party so that insurance pool balance > 0, submit trading terminated. 
+1. System saves LNL checkpoint at a time when undistributed LP fees for the market are > 0.
+1. Restart Vega, load LNL checkpoint. 
+1. The market is not restored (it doesn't exist in core i.e. it's not possible to submit orders or LP provisions to this market) (<a name="0073-LIMN-029" href="#0073-LIMN-029">0073-LIMN-029</a>) 
+1. If the market exists in the data node it is marked as settled with no settlement price info (<a name="0073-LIMN-030" href="#0073-LIMN-030">0073-LIMN-030</a>)
```

Comment:
Duplicate AC code between this and the one below unfortunately.

Reply:
Oh bugger, that'll mess up the sequencing 🤣 .